router/src/infer.rs

```diff
@@ -1553,7 +1553,9 @@ fn aggregate_ner_output_simple(
     }
     let mut new_ner_results = Vec::with_capacity(ner_results.len());
     for mut entity in ner_results {
-        entity.word = input[entity.start..entity.end].to_string();
+        entity.word = input[entity.start..entity.end]
+            .to_string()
+            .to_ascii_lowercase();
```

Comment:
Needed to exactly match HF output

Reply:
Can you add this comment to the code here to explain the lowercasing?